### DISCUSSION TITLE
[Scam Alert] AI Tradies     ای آی تریدز

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#45] [Announcements]
<div dir="rtl">

**Project Name:** 
AItradies 

**نام پروژه:** 

ای آی تریدز
**نماد پروژه:**
ATC
**اتفاقی که افتاده است:**

توکنی است که بروی شبکه استالر تولید و منتشر شده. به نام ATC
توکن ATC تنها در صرافی AiTrades معامله می شود.
ادعای این پروژه این است که به کمک هوش مصنوعی ربات ها و تبادل در بازار برای کاربران سود تضمین شده تولید می کند.

شاخص های مهم جهت بررسی پروژه های با موضوعات مشابه:
ولت: ندارد.
پیج گیتهاب: ندارد
وایت پیپر: ندارد
ادعا این است که یک ربات، با الگوریتم سود می دهد.
گزارش ها حاکی از آن است که تنها یک فقره بیش از 200 هزار دلار بروی این پروژه سرمایه گذاری کرده.
شفافیت؟
چطور می توان به پروژه ای که ادعا می کند با هوش مصنوعی ثروت خلق می کند، اما از طرفی نه گیتهاب برای رجوع و بررسی کد و نه وایت پیپر شفافی جهت توضیح نحوه درآمد دارد، اعتماد کرد.  
در بازار رمزارزها بدلیل نوسانات شدید و تاثیر اخبار بر قیمت لحظه ای توکن ها، سود تضمین شده بیشتر از 2 الی 3 درصد در ماه بسیار نامتعارف و خطرناک است.
با این وضعیت این امکان وجود دارد که حتی رباتی در کار نباشد و تنها با اضافه شدن افراد جدید، این پروژه به حیات خود ادامه دهد. در غیر این صورت تامین سود تضمین شده برای اعضای قبلی امکان پذیر نخواهد بود.
در سایت، بخش سوالات متداول بجای توضیح و رفع ابهام در مورد پروژه سوالاتی مانند "بیت کوین چیست؟" "هوش مصنوعی چیست؟" را پاسخ داده. این نشان دهنده این است که پروژه هیچ پشتوانه ای برای ادعای خود ندارد و صرفا تلاش کرده صفحات سایت را پر کند.
چطور می شود کسی نداند هوش مصنوعی چیست و مخاطب پروژه ای باشد که ادعای توسعه هوش مصنوعی دارد؟ این خود تناقض در عملکرد است. در واقع مخاطبان این پروژه افراد تازه واردی هستند که به دلیل کمبود آگاهی و نا آشنا بودن با فضا، فریب چند کلمه مشهور در این حوزه را می خورند و اعتماد می کنند. 
چرا کسی که نمی داند بلاکچین و بیت کوین چیست باید از این پروژه و خدمات آن استفاده کند؟
توکن ATC در صرافی اختصاصی مرتبط به پروژه، معامله می شود یکی از نکات عجیب آنکه، چطور می شود در بازار رمزارز که نوسانات در ثانیه اتفاق می افتد قیمت این توکن در سایت همیشه ثابت نمایش می شود؟ و این نشانه فریب کاری است.
 
با بررسی توکن بروی شبکه استالر می توان به گذاره های زیر رسید:
نحوه تولید و توزیع توکن:
از توکن ATC ،  20 میلیون تولید شده که فقط 11 میلیون آن در اختیار 1 آدرس است.
تمام توکن های ATC در حدود 875 آدرس وجود دارد. 
با توجه به کاربران زیادی که در فضای مجازی هم مشهود است و همنیطور رصد تعداد کاربران در تلگرام می توان دریافت که در این پروژه احتمالا خالی فروشی صورت گرفته به این معنا که خریداران توکن ATC بروی شبکه مالک توکن نشده اند و سوابق و اطلاعات فقط در حساب داخلی شرکت ثبت شده است.  
بیش از 50 درصد توکن ها در اختیار یک آدرس است و امکان تقلب در قیمت و ایجاد نوسان به منظور سود بیشتر را برای دارندگان اصلی توکن فراهم می کند. از رو پروژه به شدت خطرناک است و سرمایه های مردم به اراده یک فرد می تواند نابود شود. 
پلن ها سرمایه گذاری 2 ساله است و امکان برداشت را به کاربر نمی دهد. خاطی هم با خیال راحت از فرصت دو ساله ای که ایجاد شده سو استفاده می کند و پول نقد را دریافت و به صورت اقساط خرد به کاربر بر می گرداند.
بررسی عملکرد: در بسیاری از مواقع سایت پروژه باگذاری نمی شود و نشان دهدنه بی اهمیتی موضوع برای مدیرانی پروژه است که میلیون ها دلار از سرمایه کاربران را در دست دارند.  
تمامی موارد بالا حاکی از یک پروژه پر ریسک و بسیار خطرناک است.

**قوانینی که نقض شده است:**

نقض ماده 44 قانون اجرای سیاستهای کلی اصل چهل و چهارم (44) قانون اساسی 

**نوع هشدار:**
کلاهبرداری

**نوع کلاهبرداری:** 
پانزی

**وضعیت کلاهبرداری:**

جاری

**نام کلاهبرداران:**

**نام شرکت(ها):**
ای آی تریدز
**آدرس وب سایت(ها):**

وب سایت:  https://aitrades.com
 
**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**
تمامی موارد بالا حاکی از یک پروژه پر ریسک و بسیار خطرناک است.

**Comment by LmnterixCo** [comment]
اعصای این پروژه شناسایی شده و در حال پیگیری هست
یکی از اعضا داخل ایران دستگیر شده.

**Comment by FeelsDejavu** [comment]
https://www.instagram.com/p/Cc-6JVKr8vz/?igshid=YmMyMTA2M2Y=

پیج شاکی ها که اسم ادمین ها هست
